app/templates/index.html

```diff
@@ -49,31 +72,56 @@ <h3 class="panel-title">Most Failed Packages</h3>
+             				  Show in Tabular form             				</button>
+             				<div class="collapse" id="collapseExample">
+             				  	<table class="table table-hover">
+             				  		<thead>
+             				  	     	<tr>
+             				  	        	<th>Name</th>
+             				  	        	<th>Version</th>
+             				  	        	<th>Count</th>
+             				  	      	</tr>
+             				  	    </thead>
+             				  	    <tbody>
+             				  	    	{% for failed_package in response['most_failed_packages'] %}
+             				  	      		<tr>
+             				  	        		<td>{{failed_package['name']}}</td>
+             				  	        		<td>{{failed_package['version']}}</td>
+             				  	        		<td>{{failed_package['count']}}</td>
+             				  	      		</tr>
+             				  	      	{% endfor %}
+             				  	    </tbody>
+             				  	</table>
+             				    
+             				</div>
+             				<br /> <br />
              				<div class="form-group">
-	 			<label for="sel1">More details :</label>
-	  			<form action="/view/detail/" method="GET">
-		  			<select class="form-control" id="sel1" name="package_detail">
-		  				<option>Select Package</option>
-		    			{% for failed_package in response['most_failed_packages'] %}
-				    		<option value="{{failed_package['name']}}|{{failed_package['version']}}">{{failed_package['name']}} ({{failed_package['version']}})</option>
-				    			{% endfor %}
-				  	</select>
-					<div class="checkbox">
-		  				{% if not show_all%}																				  			  		<label><input type="checkbox" name="all_workshops" value="True">All workshops</label>
-				  		{% else %}
-				  	  		<input type="hidden" value="True" name="all_workshops">
-					  	{% endif %}
-				  	</div>
-					{% if not show_all %}
-		  				<input type="hidden" value="{{workshop_name}}" name="workshop_name">
-		  			{% endif %}
+	 							<label for="sel1">More details :</label>
+	  							<form action="/view/detail/" method="GET">
+		  							<select class="form-control" id="sel1" name="package_detail">
+		  								<option>Select Package</option>
+		    							{% for failed_package in response['most_failed_packages'] %}
+				    						<option value="{{failed_package['name']}}|{{failed_package['version']}}">{{failed_package['name']}} ({{failed_package['version']}})</option>
+				    					{% endfor %}
+				  					</select>
+									<div class="checkbox">
+						  				{% if not show_all%}																				  			  		<label><input type="checkbox" name="all_workshops" value="True">All workshops</label>
+								  		{% else %}
+								  	  		<input type="hidden" value="True" name="all_workshops">
+									  	{% endif %}
+				  					</div>
+									{% if not show_all %}
+						  				<input type="hidden" value="{{workshop_name}}" name="workshop_name">
+						  			{% endif %}
```

Comment:
Yes yes, I do convert tab to spaces. Just forgot to do it here as well. Will do it.